src/common/eeprom.c

```diff
@@ -48,6 +48,28 @@ const uint8_t eeprom_map_v1[] = {
     VARIANT8_UI8,
     VARIANT8_UI8,
     VARIANT8_UI8,
+    VARIANT8_UI32, // EEVAR_CONNECT_IP    X.X.X.X address encoded in uint32
```

Comment:
coordinate this with the upcoming eeprom improvements from @XPila

Reply:
we are in contact, as these functions ready we will migrate